replicate_manifest_config.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ] || [ "$#" -ne 2 ]; then
+  echo "------------------------------------------------------------------------------"
+  echo "Usage - replicate_manifest_config <source_manifest> <target_manifest>"
+  echo ""
+  echo "e.g., ./replicate_manifest_config.sh internalstaging.datastage.io/manifest.json gitops-qa/jenkins-dcp.planx-pla.net/manifest.json"
+  echo ""
+  echo "The script pulls data from a source manifest and replaces the versions & dictionary of the target manifest"
+  echo "-------------------------------------------------------------------------------"
+  exit 0;
+fi;
+
+svcs=(arborist fence indexd aws-es-proxy peregrine pidgin revproxy sheepdog portal fluentd spark tube manifestservice wts guppy sower hatchery ambassador)
+
+# TODO: add "ssjdispatcher" "quay.io/cdis/ssjdispatcher:master" when modifying DEV/QA manifests 
+
+# source manifest
+# e.g., internalstaging.datastage.io/manifest.json
+src_manifest="$1"
+echo "fetching source manifest from: https://raw.githubusercontent.com/uc-cdis/cdis-manifest/master/${src_manifest}"
+versions_and_dict=$(curl -s "https://raw.githubusercontent.com/uc-cdis/cdis-manifest/master/${src_manifest}" | jq '.versions, .global.dictionary_url')
```

Comment:
it would be good to check first if the source manifest exists locally, and use that.  The source manifest may not necessarily be in `cdis-manifest` depending on what you are doing
```
if [[ -f "$src_manifest" ]]; ...
```

Reply:
I was thinking about merging version `0.0.1` with a few TODOs to help us move forward with the release mgmt effort.

Then we can collaboratively optimize the script later.